src/test/ui/const-generics/min_const_generics/type_and_const_defaults.rs

```diff
@@ -0,0 +1,14 @@
+// check-pass
+#![crate_type = "lib"]
+#![feature(const_generics_defaults)]
+#![allow(incomplete_features)]
+
+struct Both<T=u32, const N: usize=3> {
```

Comment:
Do we still require defaulted const parameters to follow defaulted type parameters?

Reply:
no, there's no restriction here `Both<const N: usize = 3, T = u32>`, `Both<const N: usize, T = u32>` and `Both<const N: usize, T = u32>` should also work

Reply:
@JulianKnodt: can we make sure we have tests for this here too? Along with instantiations of the structs to show the order is being respected in the arguments.

Reply:
@lcnr for const-generics, as it is I don't think we permit for the `Both<const N: usize=3, T>` case yet

Reply:
yeah, we don't permit anything like `Both<const N: usize = 3, T>`. But I also didn't mention that case, did I?

Reply:
ah woops I misread, that's my bad